OMCompiler/Compiler/SimCode/SimCodeVar.mo

```diff
@@ -70,11 +70,12 @@ uniontype SimVars "Container for metadata about variables in a Modelica model."
     list<SimVar> sensitivityVars "variable used to calculate sensitivities for parameters nSensitivitityParameters + nRealParam*nStates";
     list<SimVar> dataReconSetcVars;
     list<SimVar> dataReconinputVars;
+    list<SimVar> modelDescriptionVars "variables list which are used in FMI-2.0 modelDescription.xml";
```

Comment:
`modelDescriptionVars` is probably not needed with the new approach, right?

Reply:
@lochel This is needed, because it is easier to update the special fmi index and also it is easier to update the dependencies in model structure,  and this list will only contains the modeldescription varlist to be exported